showpass/Showpass-E2E-tests
# Add the 'tag-smoke.yml' workflow

**Description:**
This workflow will be used to execute specs labeled as 'smoke' tests.

**How to test:**
Run this workflow in the GitHub Actions and ensure the right tests get executed.
